config/migrations/20220825163000-case-and-decisionmaking-flow-migration-kanselarij.sparql

```diff
@@ -0,0 +1,106 @@
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>
+
+WITH <http://mu.semte.ch/graphs/organizations/kanselarij>
+INSERT {
+  ?besluitvormingsaangelegenheid a besluitvorming:Besluitvormingsaangelegenheid ;
+    mu:uuid ?uuid .
+  ?dossier dossier:Dossier.isNeerslagVan ?besluitvormingsaangelegenheid .
+}
+WHERE {
+  ?dossier a dossier:Dossier .
+  FILTER NOT EXISTS {
+    ?dossier dossier:Dossier.isNeerslagVan ?existingBesluitvormingsaangelegenheid .
+  }
+
+  {
+    FILTER NOT EXISTS {
+      ?publicatieaangelegenheid a pub:Publicatieaangelegenheid .
+      ?publicatieaangelegenheid dossier:behandelt ?dossier .
+    }
+  } UNION {
+    FILTER EXISTS {
+      ?agendapunt a besluit:Agendapunt .
+      ?agendapunt ^besluitvorming:heeftOnderwerp ?behandelingVanAgendapunt .
+      ?behandelingVanAgendapunt besluitvorming:heeftBeslissing ?beslissingsActiviteit .
+      ?beslissingsActiviteit ^dct:subject ?publicatieaangelegenheid .
+      ?publicatieaangelegenheid dossier:behandelt ?dossier .
+    }
+  }
+
+  BIND(STRUUID() AS ?uuid)
```

Comment:
Genereert dit effectief verschillende UUIDs per instance? Dit lijkt mij een case zoals https://github.com/openlink/virtuoso-opensource/issues/515. Workaround is voorzien in https://github.com/openlink/virtuoso-opensource/issues/515#issuecomment-456848368

Reply:
Double checked to be sure, seems like it does generate different UUIDs.
![image](https://user-images.githubusercontent.com/22411874/188720957-b102bc82-fb4a-4fb6-8975-149adba0661a.png)

Each migration took about ~40 seconds to run on a copy of the ACC database, for reference.